src/groups/mqb/mqbblp/mqbblp_remotequeue.cpp

```diff
@@ -1176,12 +1180,17 @@ void RemoteQueue::onAckMessageDispatched(const mqbi::DispatcherAckEvent& event)
     if (d_state_p->isAtMostOnce()) {
         // Consider this a "non-soft" ACK for all previous broadcasted PUTs.
         size_t numErased = 1 + erasePendingMessages(it);
+        d_expiredPutNum += numErased;
 
         erasePendingMessage(it);
 
-        BALL_LOG_INFO << d_state_p->uri() << ": erased window of " << numErased
-                      << " cached broadcasted PUTs upon "
-                      << bmqt::AckResult::toAscii(ackResult);
+        if (d_throttledCachedPutMessages.requestPermission()) {
+            BALL_LOG_INFO << d_state_p->uri() << ": erased window of "
+                          << numErased << " (" << d_expiredPutNum
```

Comment:
`d_expiredPutNum` is more like "total erased broadcast PUTs".  But what is the meaning of this counter; why do we need it?

Reply:
If we are throttling logs, we still might want to know how many we of these we skipped. Might be useful for debug, but also I'm okay to remove it.